Makefile

```diff
@@ -0,0 +1,13 @@
+GOROOT := $(shell go env GOROOT)
+
+.PHONY: serve
+all: help
+help:
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
I agree with @saljam's suggestion -- just do the build. I'd also just like to note that adding colors is shell dependent, and this color choice is unreadable on my light-background terminal, and makes grepping harder. (...and sends garble out on some more obscure systems that I use, but I accept that's a bit more of a niche case).

The command itself can also be simplified:

     @grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'

Could be:

       @awk -F':[^#]*##' '/^[a-zA-Z0-9_]+:.*##/{print $$1"\t"$$2}' $(MAKEFILE_LIST) | sort

Reply:
Thanks. Changed